packages/autocomplete-js/src/createAutocompleteDom.ts

```diff
@@ -101,12 +101,6 @@ export function createAutocompleteDom<TItem extends BaseItem>({
     getInputProps: propGetters.getInputProps,
     getInputPropsCore: autocomplete.getInputProps,
     autocompleteScopeApi,
-    onDetachedEscape: isDetached
-      ? () => {
-          autocomplete.setIsOpen(false);
-          setIsModalOpen(false);
-        }
-      : undefined,
```

Comment:
This was no longer needed because the logic in `onDetachedEscape` has been covered by [this](https://github.com/algolia/autocomplete/blob/8fa038b914a1b76270a106f5fe2b223aa657d6ae/packages/autocomplete-js/src/autocomplete.ts#L239) since https://github.com/algolia/autocomplete/pull/556.

This is likely stale code from the first [Autocomplete Touch (now Detached Mode) PR](https://github.com/algolia/autocomplete/pull/379).